File: boj/Implemenation/17210/17210.cpp
#include <bits/stdc++.h>
#define ll long long
using namespace std;
ll n;
int way;
int main(){
    cin >> n >> way;
    if(n > 5) {
        cout << "Love is open door";
    }
    else {
        for(int i = 1; i < n; i++){
            way = 1 - way;
            cout << way << '\n';
        }
    }
}

File: boj/Implemenation/17210/17210.rs
use std::io;

fn main() {
    let mut line = String::new();
    io::stdin().read_line(&mut line).expect("wrong io");
    let door_num = line.trim().parse::<i64>().unwrap();

    let mut line = String::new();
    io::stdin().read_line(&mut line).expect("wrong io");
    let mut way = line.trim().parse::<i64>().unwrap();

    if door_num > 5 {
        print!("Love is open door");
    } else {
        for _ in 1..door_num {
            way = 1 - way;
            println!("{}", way);
        }
    }
}
